vehicle-syntax/src/Vehicle/Syntax/AST/Decl.hs

```diff
@@ -194,3 +194,6 @@ instance Pretty Annotation where
 
 isProperty :: [Annotation] -> Bool
 isProperty anns = AnnProperty `elem` anns
+
+isTensor :: [Annotation] -> Bool
+isTensor anns = AnnTensor `elem` anns
```

Comment:
Hmm so the original `isProperty` wasn't a great name, and I'm not sure this is either as it's really just testing annotations, and the declaration definitely isn't a record! Could we refactor to both these names to `isAnnotatedAsProperty` and `isAnnotatedAsTensor`?